warehouse/utils/utils.go

```diff
@@ -168,6 +168,20 @@ type QueryResult struct {
 	Values  [][]string
 }
 
+type PendingEventsRequest struct {
```

Comment:
lets add a T at the end `PendingEventsRequestT` ? just check once if that's what we are doing for other structs too